src/ontology/mondo-edit.obo

```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
 xref: OMIM:620690 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000158 {source="OMIM:620690"} ! developmental dysplasia of the hip
 
+[Term]
+id: MONDO:0958083
+name: conjoined twins
+xref: Orphanet:647916 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019755 {source="Orphanet:647916"} ! developmental defect during embryogenesis
+
+[Term]
+id: MONDO:0958091
+name: cleft palate-congenital heart defect-intellectual disability syndrome
+xref: Orphanet:652519 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-4142-7153"} ! syndromic disease
+is_a: MONDO:0015159 {source="Orphanet:652519"} ! multiple congenital anomalies/dysmorphic syndrome-intellectual disability
+
+[Term]
+id: MONDO:0958094
+name: adult-onset progressive leukoencephalopathy-early-onset deafness
+def: "A rare genetic neurological disorder characterized by congenital or early-onset sensorineural deafness and adult-onset progressive leukoencephalopathy. Progressive cognitive impairment and behavioral abnormalities are observed in the second or third decade of life, sometimes preceded by mild developmental delay and learning difficulties. Visual impairment in adult age has been reported. No central nervous system calcification is reported." [Orphanet:652532]
+xref: Orphanet:652532 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019046 {source="Orphanet:652532"} ! leukodystrophy
+
+[Term]
+id: MONDO:0958106
+name: congenital insensitivity to pain syndrome, marsili type
```

Comment:
this is something that needs to be done as part of this process - the labels need to be reviewed for capitalization, this should probably be Marsili not marsili

Reply:
Thank you for catching this! I missed the capitals. :-)